corehq/apps/accounting/models.py

```diff
@@ -1964,13 +1965,24 @@ class Meta(object):
 
     _pdf = None
 
+    @property
+    def emailed_to(self):
+        return ','.join(self.emailed_to_list)
+
     @property
     def recipients(self):
-        return self.emailed_to.split(',') if self.emailed_to else []
+        return self.emailed_to_list
 
     @recipients.setter
     def recipients(self, emails):
-        self.emailed_to = ','.join(emails)
+        self.emailed_to_list = emails
```

Comment:
now that there's no transformation necessary, I think you can drop the `setter` and `getter` business and just switch things which call `billing_record.recipients` to `billing_record.emailed_to_list`.